tests/runtime/array

```diff
@@ -142,3 +142,9 @@ RUN {{BPFTRACE}} -e 'struct A { int x[4]; } uprobe:./testprogs/array_access:test
 EXPECT Result: 1
 TIMEOUT 5
 AFTER ./testprogs/array_access
+
+NAME array of pointers element access
+PROG struct A { int *x[4]; } uprobe:./testprogs/array_access:test_ptr_array { @x = *((struct A*)arg0)->x[1]; exit(); }
```

Comment:
Just to match the naming in array_access.c (I got confused on the first read of this test):
```suggestion
PROG struct C { int *z[4]; } uprobe:./testprogs/array_access:test_ptr_array { @x = *((struct C*)arg0)->z[1]; exit(); }
```